### PR TITLE
fix test episode and marlgrid configs

### PR DESCRIPTION
**Pull request by hnekoeiq**

### CODE REVIEW

File: hive/configs/marlgrid/rainbow.yml [hunk 1]
```diff
@@ -106,6 +107,7 @@ agents:
           steps: 10000
       min_replay_history: 500
       batch_size: 32
+      device: 'cpu'
```
Comment (dapatil211) [inline hive/configs/marlgrid/rainbow.yml]: Why was this added?

**Review by dapatil211 — COMMENTED** [review]
For future, try to separate out changes corresponding to different things into different PRs. Also why did you add a device, and why cpu?

File: hive/configs/marlgrid/rainbow.yml (hunk 1 above)
Comment (hnekoeiq) [reply hive/configs/marlgrid/rainbow.yml]: I didn't notice that now the agent itself recognizes the device. Will remove it.

Comment (hnekoeiq) [reply hive/configs/marlgrid/rainbow.yml]: Actually, since marlgrid experiments should be run on gpu anyway, I changed it to cuda.

**Review by dapatil211 — APPROVED** [review]

File: hive/runners/base.py [hunk 2]
```diff
@@ -156,7 +156,7 @@ def run_testing(self):
         self.train_mode(False)
         aggregated_episode_metrics = self.create_episode_metrics().get_flat_dict()
         episodes = 0
-        while episodes <= self._test_episodes:
+        while episodes < self._test_episodes:
```
Comment (sriyash421) [inline hive/runners/base.py:159]: This was updated by a recent pr, so could you rebase and remove the conflicts? @hnekoeiq

**Review by sriyash421 — CHANGES_REQUESTED** [review]

**Comment by hnekoeiq** [comment]
Closing this since #281 fixed that.
